spec/abstractops.html

```diff
@@ -1071,11 +1061,9 @@ <h1>ISO 8601 grammar</h1>
       DateSpecYearMonth :
           DateYear `-`? DateMonth
 
-      TwoDashes :
-          `--`
-
       DateSpecMonthDay :
-          TwoDashes? DateMonth `-`? DateDay
+          `--` DateMonth `-`? DateDay
+          DateMonth `-`? DateDay
```

Comment:
Note https://github.com/tc39/proposal-temporal/issues/1800; but I think this is still in line with that.